packages/test/test-end-to-end-tests/src/test/container.spec.ts

```diff
@@ -638,6 +642,12 @@ describeNoCompat("Container", (getTestObjectProvider) => {
 			1,
 			"ContainerRuntime should send dispose event on container dispose",
 		);
+
+		// Also check logged events
+		mockLogger.assertMatch([
```

Comment:
why not use itExpects

Reply:
Wasn't sure it worked for non-errors.  I'll switch to it.

Reply:
Also, I like being able to check the logs at different points (see following test).  But not a big deal I think